packages/@okta/vuepress-site/docs/guides/embedded-siw/main/index.md

```diff
@@ -0,0 +1,428 @@
+---
+title: Embedded Okta Sign-In Widget fundamentals
+---
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+This guide explains authentication fundamentals using JavaScript and the embedded Okta Sign-In Widget, and provides a simple one-page SPA application to demonstrate a sign-in use case.
+
+---
+
+**Learning outcomes**
+
+* Understand how to implement basic sign-in using JavaScript and the embedded Okta Sign-In Widget.
+* Understand basic installation and code configurations using JavaScript.
+* Implement a simple SPA use case and sign a user in to the application.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup/)
+* Basic knowledge of building front-end JavaScript applications
+
+**Sample code**
+
+* Sample code is provided in the following section: [Sign In and display user's email](#sign-in-and-display-user-s-email)
+
+---
+
+## About the embedded Okta Sign-In Widget
+
+The Okta Sign-In Widget is a JavaScript library that gives you a fully-featured and customizable sign-in experience, which you can use to authenticate users on any website.
+
+Okta uses the Widget as part of its normal sign-in page. If you would like to fully customize the Widget, then you need to host it yourself. This guide walks you through the [installation process](#installation) for the Widget, as well as [a few common use cases](#sign-in-widget-use-cases) for the Widget and how to implement them. The full Widget reference can be found [on GitHub](https://github.com/okta/okta-signin-widget#okta-sign-in-widget).
+
+A simple working code example is also included to demonstrate a common sign-in use case. See [Sign In and display user's email](#sign-in-and-display-user-s-email)
+
+> A version of the Widget that you can edit in real time is available on the [Okta Developer Documentation site](https://developer.okta.com/live-widget/).
+
+<img src="/img/okta-sign-in-javascript.png" alt="Screenshot of basic Okta Sign-In Widget" width="400">
+
+## Installation
+
+The first step is to install the Widget. You have two options: linking out to the Okta CDN, or installing locally through `npm`.
+
+### CDN
+
+To use the CDN, include this in your HTML:
+
+```html
+<!-- Latest CDN production JavaScript and CSS -->
+<script src="https://global.oktacdn.com/okta-signin-widget/-=OKTA_REPLACE_WITH_WIDGET_VERSION=-/js/okta-sign-in.min.js" type="text/javascript"></script>
+<link href="https://global.oktacdn.com/okta-signin-widget/-=OKTA_REPLACE_WITH_WIDGET_VERSION=-/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
+```
+
+More info, including the latest published version, can be found in the [Okta Sign-In Widget SDK](https://github.com/okta/okta-signin-widget#using-the-okta-cdn).
+
+### npm
+
+```bash
+# Run this command in your project root folder.
+npm install @okta/okta-signin-widget@-=OKTA_REPLACE_WITH_WIDGET_VERSION=-
```

Comment:
Will the widget always be the latest version?

Reply:
Yes, but we do need to update a config page on our site with the latest version release (it picks up point release). I've done this with PR 2996: https://github.com/okta/okta-developer-docs/pull/2996